libexec/slurm/sbatch.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+#-------------------------------------------------------------------------------
+# NOTE:
+#
+# Please make installation specific modifications within this file.
+# The last line printed to STDOUT for a successful deployment must be the
+# response from sbatch:
+#
+# $ sbatch my-script
+# > Submitted batch job XX
+#-------------------------------------------------------------------------------
+sbatch "$1"
```

Comment:
What's the purpose of this file?

Reply:
This is a legacy implementation from `flight-job-script-api`.

The `submit.sh` contains a bunch of boiler plate code concerning the JSON response the tool is expecting. This means the `submit.sh` system managed and does not support edits.

So a compromise to this was to create the `sbatch.sh` which is packaged as as config file and can be modified with installation specific details.

Reply:
What installation specific details do you imagine would be needed?  Perhaps this would be better renamed to `sbatch_wrapper.sh` to communicate its purpose.

Reply:
Renamed here: https://github.com/openflighthpc/flight-job/pull/3/commits/da2842f1aea2a1d5fd84f351f2865fc28cedc472

There could be various flags that a sysadmin might want to set without worrying about updating every `template`/`script`. The most likely candidates are `--cluster`, `--priority`, and possible `--time`.

Or they may want to preform some bespoke action depending on who the user is. The wrapper provides a useful integration point getting caught up in the `submit.sh` JSON handling.

In general a more flexible "submit" mechanism may be preferred, however the requirements have not been established for this.